addons/website_sale/controllers/main.py

```diff
@@ -926,3 +954,23 @@ def tracking_cart(self, **post):
             }
             ret['lines'] = self.order_lines_2_google_api(order.order_line)
         return ret
+
+    # link to the backend for portal users
+    @http.route('/shop/backend', type='http', auth="public")
```

Comment:
Special last minute request from @antonylesuisse : put a link that can lead customers to the backend if they wish (i.e. for big clients with hundreds of records)